tsconfig.json

```diff
@@ -25,7 +25,7 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2017",
+    "target": "es2019",
```

Comment:
wasn't sure about this one. I'm sure the code can be tweaked to avoid using `Object.fromEntries`